server/utils/urlGenerator.ts

```diff
@@ -183,3 +183,10 @@ export const deletePerson = {
     start: '/',
   },
 }
+
```

Comment:
This should probably be called homepage so as to be consistent with the others in this file?